src/thx/bigint/Bigs.hx

```diff
@@ -223,8 +225,8 @@ class Bigs {
 			difference = a[i] + carry;
 			carry = Math.floor(difference / BASE);
 			// Chrome resolves -1 % 1 to -0 and -0 < 0 == true, Std.int fixes this with -0 | 0 = 0
-			remainder = Std.int(difference % BASE);
-			r[i] = difference < -BASE ? (remainder < 0 ? remainder + BASE : remainder) : difference;
+			// Code below normalize result. Returns always integer in range [0, BASE] even for negative input.
```

Comment:
"Code below" is noise, it is expected that comments refer to the following line. Also "Always returns" not "returns always", and it is not actually a return, but an assignment
```suggestion
			// Normalize to integer in range [0, BASE], even for negative input.
```